lib/JsModule.js

```diff
@@ -44,13 +46,15 @@ export default class JsModule {
     relativeFilePath,
     stripFileExtensions,
     stripFromPath,
+    variableName,
```

Comment:
I needed to be able to call `toImportStatement` in a context where I didn't have access to the variable name.